src/Configurator/DockerComposeConfigurator.php

```diff
@@ -259,7 +259,7 @@ private function configureDockerCompose(Recipe $recipe, array $config, bool $upd
                 }
 
                 // Keep end in memory (check break line on previous line)
-                $endAt[$node] = '' !== trim($lines[$i - 1]) ? $i : $i - 1;
+                $endAt[$node] = !$i || '' !== trim($lines[$i - 1]) ? $i : $i - 1;
```

Comment:
I change your fix a bit because the previous code was putting a `-1` into `$endAt[$node]`.
Can you please confirm this works for you?

Reply:
yes, it works